tests/Session/test_commands.py

```diff
@@ -30,12 +31,36 @@
 from unittest.mock import patch
 
 import io
+import json 
+import os
 import sys
 import unittest
 
 class Test_Commands(unittest.TestCase):
     def setUp(self):
         self.commands = Commands()
+    
+    def tearDown(self):
+        try:
+            os.remove(PROFILES_PATH + SLASH + profile_name + JSON_EXTENSION)
+        except FileNotFoundError:
+            return
+    
+    # Compares contents between two profiles
+    def compareProfiles(self, profile1, profile2):
```

Comment:
Instead of creating a helper here, you could expand `__eq__` and `__ne__` so that you can just do `self.assertEqual(profile1, profile2)`